webdriver-spec.html

```diff
@@ -5602,33 +5605,54 @@
   of <a>trying</a> to <a>get a known element</a>
   with argument <var>element id</var>.
 
+ <li><p><a>Scroll into view</a> the <var>element</var>.
```

Comment:
I would like this to behave the way _Element Click_ does: We only need to scroll it into view if it is not already in view. You can steal the necessary steps from _Element Click_.